test/Errors.test.js

```diff
@@ -92,6 +92,44 @@ describe("Errors", function() {
 			done();
 		});
 	});
+	it("should warn about NoErrorsPlugin being deprecated in favor of NoEmitOnErrorsPlugin", function(done) {
+		getErrors({
+			entry: "./no-errors-deprecate",
+			plugins: [
+				new webpack.NoErrorsPlugin()
+			]
+		}, function(errors, warnings) {
+			if(errors.length === 0) {
```

Comment:
I simply took what was above in the same file! I do agree they don't really make sense though.